site/content/overview/resource-validation.md

```diff
@@ -92,6 +92,7 @@ This step catches the following cases of invalid values:
 - Valid values from the Gateway API perspective, but invalid for NGINX, because NGINX has stricter validation requirements for certain fields. These values will cause NGINX to fail to reload or operate erroneously.
 - Invalid values (both from the Gateway API and NGINX perspectives) that were not rejected because Step 1 was bypassed. Similar to the previous case, these values will cause NGINX to fail to reload or operate erroneously.
 - Malicious values that inject unrestricted NGINX config into the NGINX configuration (similar to an SQL injection attack).
+- The path prefix "/ngf_internal" for http route is reserved for internal use. Please refrain from using it for your own endpoints to avoid conflicts with internal processes.
```

Comment:
Let's add a class of errors rather than a specific error. Also, it makes sense to move it under "Valid values from the Gateway API perspective, but invalid for NGINX"
```suggestion
- Valid values from the Gateway API perspective, but invalid for NGINX Gateway Fabric, because such values
can prevent generating NGINX configuration.
```
Note that we don't need to list examples of invalid values here, having a class would be enough.

Reply:
will update